### PR TITLE
feat: add Vietnamese translations

### PR DESCRIPTION
**Pull request by kyrie25**
**Add Vietnamese translations for ~50 presences**

<details>
<summary> Proof to proving the creation/modification working </summary>
<br>

nah I'm good lmao
</details>

### CODE REVIEW

**Comment by EncryptedDev** [comment]
@ImRodry

File: websites/B/Battlefield Portal/dist/metadata.json [hunk 1]
```diff
@@ -1,50 +1,51 @@
...
+      "title": "Show title",
+      "value": true
+    },
+    {
+      "id": "desc",
+      "title": "Show description",
+      "value": true
+    }
+  ]
+}
```
Comment (ImRodry) [inline websites/B/Battlefield Portal/dist/metadata.json:51]: Please undo the formatting changes on this one

Comment (kyrie25) [reply websites/B/Battlefield Portal/dist/metadata.json:51]: Wait it's exactly the same as before though

Comment (ImRodry) [reply websites/B/Battlefield Portal/dist/metadata.json:51]: It’s very clearly not

Comment (kyrie25) [reply websites/B/Battlefield Portal/dist/metadata.json:51]: Gonna change it back first thing tomorrow morning  then, maybe VS Code is messing with me or sth

Comment (kyrie25) [reply websites/B/Battlefield Portal/dist/metadata.json]: Done(?)

File: websites/B/Battlefield Portal/dist/metadata.json [hunk 2]
```diff
@@ -47,4 +48,4 @@
         "value": true
       }
     ]
-  }
\ No newline at end of file
+  }
```
Comment (ImRodry) [inline websites/B/Battlefield Portal/dist/metadata.json:51]: Bing bong missed this also bump version

Comment (kyrie25) [reply websites/B/Battlefield Portal/dist/metadata.json:51]: Yeah I knew I forgot _something_ in my sleep. Will fix ASAP

Comment (kyrie25) [reply websites/B/Battlefield Portal/dist/metadata.json:51]: Bumped 👊 and ignore the typo plz

**Review by ImRodry — APPROVED** [review]
Author is a Vietnamese proofreader so I’d say the translations are trustworthy

Also just a tip: run `yarn ms` to fix the formatting of some files

**Review by EncryptedDev — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**